frontend/app/src/components/defi/yearn/YearnAssetsTable.vue

```diff
@@ -84,6 +87,6 @@ const vaults = computed(() => {
       <template #item.roi="{ item }">
```

Comment:
I think table from ui binds `row` as the slot prop
`:row="row"`

Reply:
> I think table from ui binds `row` as the slot prop `:row="row"`

Yes I realized there where a couple things wrong with the table change

Reply:
I updated the PR only changed this component, so that's the only thing to check again if you checked the rest.